### ISSUE TITLE
lp3 still has buildfromproperties -

### ISSUE THREAD

**Issue by rnugent3** [HydrologicEngineeringCenter/fda-statistics#29]
lp3 still has buildfromproperties - i think we should fix that.  If we allow for setting of our constituent properties (e.g. mean, stdev, and skew) we need to ensure that the other parameters we rely on are kept up to date when those properties get set.

_Originally posted by @HenryGeorgist in https://github.com/HydrologicEngineeringCenter/fda-statistics/issues/28#issuecomment-1002150782_

**Comment by HenryGeorgist** [comment]
i have an immediate fix in place:

```
            if(dist.Type==Statistics.IDistributionEnum.LogPearsonIII){
                Statistics.Distributions.LogPearson3 lp3 = (Statistics.Distributions.LogPearson3)dist;
                lp3.BuildFromProperties();
                return lp3 as Statistics.IDistribution;
            }
```
Which allows us to keep on without fixing the broader problem

**Comment by HenryGeorgist** [comment]
fixed with #58
